srcopsmetrics/entities/readme.py

```diff
@@ -32,7 +34,9 @@
 class ReadMe(Entity):
     """GitHub ReadMe entity."""
 
-    entity_schema = Schema({"name": str, "path": str, "content": str, "type": str, "license": str, "size": int,})
+    entity_schema = Schema(
+        {"name": str, "path": str, "content": str, "type": str, "license": Any(None, str), "size": int,}
```

Comment:
aren't we mixing context? this is readme class, should we have another class for that?

Reply:
hmm, that's actually true :thinking: